$OpenBSD: patch-src_FbTk_Theme_cc,v 1.1 2012/12/27 13:51:17 dcoppa Exp $

From d8cd6a928b76dbe63afa82327c51715cb1a1bdc8 Mon Sep 17 00:00:00 2001
From: Mathias Gumz <akira at fluxbox dot org>
Date: Tue, 11 Dec 2012 18:37:59 +0000
Subject: Bugfix: use expanded style/theme filename

'filename' might contain a '~' which has to be expanded to work for
regular system calls. We expanded 'filename' already but then did
not use it.

--- src/FbTk/Theme.cc.orig	Thu Dec 27 12:47:17 2012
+++ src/FbTk/Theme.cc	Thu Dec 27 12:48:00 2012
@@ -120,7 +120,7 @@ bool ThemeManager::load(const string &filename,
     StringUtil::removeFirstWhitespace(location);
     string prefix = "";
 
-    if (FileUtil::isDirectory(filename.c_str())) {
+    if (FileUtil::isDirectory(location.c_str())) {
         prefix = location;
 
         location.append("/theme.cfg");
